static/app/views/performance/transactionSummary/transactionVitals/content.tsx

```diff
@@ -66,7 +67,7 @@ function VitalsContent(props: Props) {
             {results => {
               const isMissingVitalsData =
                 !results.isLoading &&
-                allVitals.some(vital => !results.vitalsData?.[vital]);
+                shouldDisplayMissingVitalsAlert(results.vitalsData, allVitals);
```

Comment:
I think the names of these two variables should be swapped
`shouldDisplayMissingVitalsAlert` checks if there are any vitals missing, so it can be named `checkIfVitalsDataMissing` (or `isVitalsDataMissing`)
and `isMissingVitalsData` checks whether the alert should be displayed because it's also checking `results.isLoading`

Reply:
Oh good catch @udameli 

I intended to rename these but totally forgot 😅 
Initially I was planning on replacing the boolean `isMissingVitalsData` and putting `!isLoading && shouldDisplayMissingVitalsAlert` inside the conditional render, but I like your suggestion more